src/containers/TermsOfUse/index.tsx

```diff
@@ -0,0 +1,330 @@
+          Other product and business names that are mentioned on the Service may be trademarks of their respective
+          owners. We reserve all rights that are not expressly granted to you under this Agreement.
+        </p>
+
+        <h2 className="TermsOfUse__subtitle">Email may not be used to provide notice</h2>
+        <p>
+          Communications made through the Service's e-mail and messaging system, will not constitute legal notice to the
+          Company or any of its officers, employees, agents or representatives in any situation where notice to the
+          Company is required by contract or any law or regulation.
+        </p>
+
+        <h2 className="TermsOfUse__subtitle">User consent to receive communications in electronic form</h2>
+        <p>
+          For contractual purposes, you (a) consent to receive communications from the Company in an electronic form via
+          the email address you have submitted; and (b) agree that all Terms of Use, agreements, notices, disclosures,
+          and other communications that the Company provides to you electronically satisfy any legal requirement that
+          such communications would satisfy if it were in writing. The foregoing does not affect your non-waivable
+          rights. We may also use your email address, to send you other messages, including information about the
+          Company and special offers. You may opt out of such email by changing your account settings or sending an
+          email to thenewboston. Opting out may prevent you from receiving messages regarding the Company or Special
+          Offers.
+        </p>
+
+        <h2 className="TermsOfUse__subtitle">Warranty</h2>
+        <p>
+          THE SERVICE, IS PROVIDED "AS IS," WITHOUT WARRANTY OF ANY KIND. WITHOUT LIMITING THE FOREGOING, THE COMPANY
+          EXPRESSLY DISCLAIMS ALL WARRANTIES, WHETHER EXPRESS, IMPLIED OR STATUTORY, REGARDING THE SERVICE INCLUDING
+          WITHOUT LIMITATION ANY WARRANTY OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, TITLE, SECURITY,
+          ACCURACY AND NON-INFRINGEMENT. WITHOUT LIMITING THE FOREGOING, THE COMPANY MAKES NO WARRANTY OR REPRESENTATION
+          THAT ACCESS TO OR OPERATION OF THE SERVICE WILL BE UNINTERRUPTED OR ERROR FREE. YOU ASSUME FULL RESPONSIBILITY
+          AND RISK OF LOSS RESULTING FROM YOUR DOWNLOADING AND/OR USE OF FILES, INFORMATION, CONTENT OR OTHER MATERIAL
+          OBTAINED FROM THE SERVICE. SOME JURISDICTIONS LIMIT OR DO NOT PERMIT DISCLAIMERS OF WARRANTY, SO THIS
+          PROVISION MAY NOT APPLY TO YOU.
+        </p>
+
+        <h2 className="TermsOfUse__subtitle">Limitation of damages; Release</h2>
+        <p>
+          TO THE EXTENT PERMITTED BY APPLICABLE LAW, IN NO EVENT SHALL THE COMPANY, ITS AFFILIATES, DIRECTORS, OR
+          EMPLOYEES, OR ITS LICENSORS OR PARTNERS, BE LIABLE TO YOU FOR ANY LOSS OF PROFITS, USE, OR DATA, OR FOR ANY
```

Comment:
I don't understand how many of these apply to our case e.g. directors, company, employees, licensors